src/drivers/distance_sensor/ll40ls/LidarLite.h

```diff
@@ -89,20 +88,20 @@ class LidarLite
 	* range to be brought in at all, otherwise it will use the defaults LL40LS_MIN_DISTANCE
 	* and LL40LS_MAX_DISTANCE_V3
 	*/
-	void                set_minimum_distance(const float min);
-	void                set_maximum_distance(const float max);
-	float               get_minimum_distance() const;
-	float               get_maximum_distance() const;
+	void                set_minimum_distance(const float min) { _min_distance = min; }
+	void                set_maximum_distance(const float max) { _max_distance = max; }
+	float               get_minimum_distance() const { return _min_distance; }
+	float               get_maximum_distance() const { return _max_distance; }
 
-	uint32_t            getMeasureTicks() const;
+	uint32_t            getMeasureInterval() const { return _measure_interval; }
```

Comment:
Do you want to use snake_case in renaming this one?

Reply:
Yes, but let's get a basic style guide together and do it in a separate pass.